src/tree/v2/ResourceTreeDataProviderBase.ts

```diff
@@ -95,7 +93,7 @@ export abstract class ResourceTreeDataProviderBase extends vscode.Disposable imp
     }
 
     getParent(element: ResourceGroupsItem): vscode.ProviderResult<ResourceGroupsItem> {
-        return this.itemCache.getParentForItem(element);
+        return element.getParent?.();
```

Comment:
I have never seen this and am not sure how to google what it is 😅

Reply:
`maybeDefinedMethod?.()` calls `maybeDefinedMethod` if it's defined. Similar to conditionally accessing a maybe defined property with `maybeDefinedProperty?.anotherProperty`.